app/connectors/IFConnector.scala

```diff
@@ -35,6 +35,7 @@ trait IFConnector {
   val IGNORE_EMPLOYMENT = "1664"
   val UNIGNORE_EMPLOYMENT = "1699"
   val UPDATE_EMPLOYMENT = "1662"
+  val DELETE_EMPLOYMENT_DATA = "1663"
```

Comment:
I think calling this delete employment would be better since there's another api called delete employment financial data